### PR TITLE
PD-1151 Update Nextcloud Tutoiral

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates the information and screenshots in the Nextcloud turorial to reflect changes in the 24.04 UI.

Backport this change to 24.04

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Review by stavros-k — APPROVED** [review]
LGTM

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ The instructions in this article apply to SCALE 22.12.0 and later.
 
 ## Before You Begin
 
-Before using SCALE to install the Nextcloud application you need to configure TrueNAS SCALE storage for Nextcloud application to use.
+Before using SCALE to install the Nextcloud application you need to [create four datasets](#adding-nextcloud-storage) to use as storage for the Nextcloud application.
 
-If you are creating a user account to manage this application, create the user with sudo permissions enabled, or make sure the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account has sudo permissions enabled.
-If creating a new user, add this user to the dataset permissions.
+If you are creating a user account to manage this application, add the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account with sudo permissions enabled.
+If creating a new user, add this user to the dataset ACL permissions.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md:25]: Should this be something like:


```suggestion
If you are creating a user account to manage this application, enable sudo permissions and add this user to the dataset ACL permissions.

If using the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account, ensure admin sudo permissions are enabled.
```

?

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md [hunk 2]
```diff
@@ -102,68 +97,77 @@ Go to **Apps**.
...
-8. Scroll up to review the configuration settings and fix any errors or **Save** to complete the installation.
    The **Installed** screen displays with the **nextcloud** app in the **Deploying** state.
    It changes to **Running** when ready to use.
    Click **Web Portal** on the **Application Info** widget to open the Nextcloud web portal sign-in screen.
 
 {{< trueimage src="/images/SCALE/Apps/NextcloudSignInScreen.png" alt="Nextcloud Sign In Screen" id="Nextcloud Sign In Screen" >}}
 
 ## Troubleshooting Tips
+The Nextcloud apps with releases earlier than 2.0.4 are known to have issues. Use the **Upgrade** option in the SCALE UI to update your Nextcloud release to 2.0.4.
+For more information on known issues, click [here](https://ixsystemsinc.slack.com/archives/C072YK2QZKM/p1715369467541109).
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md:163]: Appears to be an incorrect link here. Maybe you meant to copy a link from slack and accidentally linked to Slack? Readers won't be able to access this.

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md [hunk 3]
```diff
@@ -102,68 +97,77 @@ Go to **Apps**.
...
 
-8. Scroll up to review the configuration settings and fix any errors or **Save** to complete the installation.
    The **Installed** screen displays with the **nextcloud** app in the **Deploying** state.
    It changes to **Running** when ready to use.
    Click **Web Portal** on the **Application Info** widget to open the Nextcloud web portal sign-in screen.
 
 {{< trueimage src="/images/SCALE/Apps/NextcloudSignInScreen.png" alt="Nextcloud Sign In Screen" id="Nextcloud Sign In Screen" >}}
 
 ## Troubleshooting Tips
+The Nextcloud apps with releases earlier than 2.0.4 are known to have issues. Use the **Upgrade** option in the SCALE UI to update your Nextcloud release to 2.0.4.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md:162]:
```suggestion
There are known issues with Nextcloud app releases earlier than 2.0.4. Use the **Upgrade** option in the SCALE UI to update your Nextcloud release to 2.0.4.
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]
Three comments for you to take a look at :)

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md (hunk 2 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md:163]: That is a bad link! I'll go get the Jira ticket link.

Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md]: Corrected the link.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**